docs/developer-guide/views.md

```diff
@@ -279,57 +566,286 @@ function App() {
 }
 ```
 
+  </TabItem>
+</Tabs>
+
+
 ### Rendering Layers in Multiple Views
 
 By default, all visible layers are rendered into all the views. This may not be the case if certain layers are designed to go into one particular view.
 
-The `Deck` class' `layerFilter` prop has access to information of the view via the `viewport` argument. It can be used to determine which layers to draw in which view: 
+The `Deck` class has a `layerFilter` prop that can be used to determine which layers to draw in which view. In the following example, two views are rendered to follow a car moving on a map: a first-person perspective from the car's dash, and a top-down perspective of the city block that it's in. We only want to render the 3D car model in the minimap because it would block the camera in the first-person view.
+
+<Tabs groupId="language">
+  <TabItem value="js" label="JavaScript">
 
 ```js
 import {Deck, FirstPersonView, MapView} from '@deck.gl/core';
-import {MeshLayer, GeoJsonLayer} from '@deck.gl/layers';
+import {SimpleMeshLayer} from '@deck.gl/mesh-layers';
+import {MVTLayer} from '@deck.gl/geo-layers';
 
-function layerFilter({layer, viewport}) {
-  if (viewport.id === 'first-person' && layer.id === 'car') {
-    // Do not draw the car layer in the first person view
-    return false;
+const deckInstance = new Deck({
+  views: [
+    new FirstPersonView({id: 'first-person'}),
+    new MapView({id: 'minimap', x: 10, y: 10, width: '20%', height: '20%'})
+  ],
+  layerFilter: ({layer, viewport}) => {
+    if (viewport.id === 'first-person' && layer.id === 'car') {
+      // Do not draw the car layer in the first person view
+      return false;
+    }
+    return true;
   }
-  return true;
+});
+
+/** Called periodically to update the map with the car's latest position */
+function updateCar(carPose) {
+  deckInstance.setProps({
+    layers: [
+      new MVTLayer({
+        id: 'base-map',
+        // ...
+      }),
+      new SimpleMeshLayer({
+        id: 'car',
+        mesh: '/path/to/model.obj',
+        data: [carPose],
+        getPosition: d => [d.longitude, d.latitude, 0],
+        getOrientation: d => [0, -d.heading * Math.PI / 180, 0]
+      })
+    ],
+    viewState: {
+      longitude: carPos.longitude,
+      latitude: carPos.latitude,
+      bearing: carPos.heading,
+      zoom: 10
+    }
```

Comment:
I think the FPV would need something like a `position: [0, -5, 0]` so that the car model is visible, but I wouldn't apply that to the MapView.